### PR TITLE
Added a new python script named String Validators.py to the Basic-Scripts folder.

### PR DESCRIPTION
**Pull request by shashwat397**
This is the program to find out if the string contains: alphanumeric characters, alphabetical characters, digits, lowercase and uppercase characters.

### CODE REVIEW

**Comment by shashwat397** [comment]
My contribution for the Basic Scripts.

File: Basic-Scripts/String Validators.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#This is the program to find out if the string  contains: alphanumeric characters, alphabetical characters, digits, lowercase and uppercase characters.
+if __name__ == '__main__':
+    s = input()
+    res = ["False", "False", "False", "False", "False"]
+    for i in s:
```
Comment (SKAUL05) [inline Basic-Scripts/String Validators.py]: Make use of meaningful variables instead of `i`.

File: Basic-Scripts/String Validators.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#This is the program to find out if the string  contains: alphanumeric characters, alphabetical characters, digits, lowercase and uppercase characters.
+if __name__ == '__main__':
```
Comment (SKAUL05) [inline Basic-Scripts/String Validators.py]: Make a separate function for your algo. In `main` only take the input from user.
Follow [PEP8 Guidelines](https://www.python.org/dev/peps/pep-0008/)

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/String Validators.py (hunk 2 above)
Comment (shashwat397) [reply Basic-Scripts/String Validators.py]: Updated String Validators.py

File: Basic-Scripts/String Validators.py (hunk 1 above)
Comment (shashwat397) [reply Basic-Scripts/String Validators.py]: Updated String Validators.py
please review the code.

**Comment by SKAUL05** [comment]
@shashwat397 Raise separate PRs for separate programs.
Don't mix Image Processing and Basic Scripts in a single PR.

**Pull request closed without merging**